.travis.yml

```diff
@@ -165,3 +167,22 @@ script:
 
   - 'if [[ "$variant" != "" ]]; then ccache --show-stats; fi'
 
+  - cd test
+  - 'if [ $arch == "arm" ];
+    then
+      echo "using gcc : arm64 : aarch64-linux-gnu-gcc : <cflags>-std=c11 <cxxflags>-std=c++11 <cxxflags>-fsigned-char <linkflags>-lstdc++ <linkflags>-lm ;" >> ~/user-config.jam;
+      wget http://releases.linaro.org/components/toolchain/binaries/latest-5/aarch64-linux-gnu/gcc-linaro-5.3-2016.02-x86_64_aarch64-linux-gnu.tar.xz;
+      tar xf gcc-linaro-5.3-2016.02-x86_64_aarch64-linux-gnu.tar.xz;
+      export PATH=${PWD}/gcc-linaro-5.3-2016.02-x86_64_aarch64-linux-gnu/bin:${PATH};
+      aarch64-linux-gnu-gcc --version;
+      wget -O boost.zip http://pilotfiber.dl.sourceforge.net/project/boost/boost/1.55.0/boost_1_55_0.zip;
+      unzip -qq boost.zip;
+      export BOOST_ROOT=$PWD/boost_1_55_0;
+      sudo apt-get install -y qemu-user-static debootstrap;
+      sudo debootstrap --variant=minbase --arch arm64 --foreign --include=build-essential testing rootfs;
+      sudo cp /usr/bin/qemu-aarch64-static rootfs/usr/bin/;
+      sudo chroot rootfs /debootstrap/debootstrap --second-stage;
+      sudo chroot rootfs mount -t proc none /proc;
```

Comment:
One thing I think would be nice would be to split this up into two parts, and do all the installation and set-up under the "install" travis script. That way all these logs will get collapsed and the test output will only show the build and test output from bjam.